eclair-core/src/main/scala/fr/acinq/eclair/package.scala

```diff
@@ -74,15 +74,10 @@ package object eclair {
   def feerateKw2Byte(feeratePerKw: Long): Long = feeratePerKw / 250
 
   /**
-   * why 253 and not 250 since feerate-per-kw is feerate-per-kb / 250 and the minimum relay fee rate is 1000 satoshi/Kb ?
+   * why 253 and not 250 since feerate-per-kw should be feerate-per-kvb / 4 and the minimum relay fee rate is 1000 satoshi/Kvb ?
    *
-   * because bitcoin core uses neither the actual tx size in bytes or the tx weight to check fees, but a "virtual size"
-   * which is (3 * weight) / 4 ...
-   * so we want :
-   * fee > 1000 * virtual size
-   * feerate-per-kw * weight > 1000 * (3 * weight / 4)
-   * feerate_per-kw > 250 + 3000 / (4 * weight)
-   * with a conservative minimum weight of 400, we get a minimum feerate_per-kw of 253
+   * because bitcoin core uses neither the actual tx size in bytes nor the tx weight to check fees, but a "virtual size"
+   * which is (weight + 3) / 4 ...
```

Comment:
lgtm, except I'd rather keep the explanation for why 253 was selected, why remove it ?

Reply:
I explained it in the commit message :

> The calculus below was then based upon this mistake so I removed it completely.

With the corrected formula, the computation doesn't hold anymore :/

Reply:
> The calculus below was then based upon this mistake so I removed it completely.

No it wasn't, the explanation and the computation are correct, there was just a typo, imho it's better to keep them and fix it.

```
fee > 1000 * virtual size	
feerate-per-kw * weight > 1000 * (3 * weight / 4) <-- typo here, should be (3 + weight) / 4 
feerate_per-kw > 250 + 3000 / (4 * weight)	<-- but not here
with a conservative minimum weight of 400, we get a minimum feerate_per-kw of 253
```

Reply:
Ok, I overlooked it. Thanks !